src/mscorlib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -566,7 +566,7 @@ internal static unsafe int CompareOrdinalIgnoreCase(ReadOnlySpan<char> strA, Rea
                 char* b = bp;
 
                 // in InvariantMode we support all range and not only the ascii characters.
-                char maxChar = (char) (GlobalizationMode.Invariant ? 0xFFFF : 0x80);
+                char maxChar = (char) (GlobalizationMode.Invariant ? 0xFFFF : 0x7F);
```

Comment:
Could this be computed just once in a static ctor? It's just accommodate, but this code is called a lot. (I can't see the context right now so this may be off.)

Reply:
@benaadams, thoughts?

Reply:
@jkotas pointed out it already is https://github.com/dotnet/coreclr/pull/16535#discussion_r170400319

Reply:
I was referring to the value, not GlobalizationMode. Ie put either 0xFFFF or 0x7F in a readonly static field. But seems unlikely that would be any faster than a conditional check each time here.